app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -129,16 +148,16 @@ export default {
       return (
         this.inReplyTo?.id &&
         !this.isPrivate &&
-        this.inboxHasFeature(INBOX_FEATURES.REPLY_TO) &&
-        !this.is360DialogWhatsAppChannel
+        inboxHasFeature(this.inbox, INBOX_FEATURES.REPLY_TO) &&
+        !is360DialogWhatsAppChannel(this.inbox)
       );
     },
     showRichContentEditor() {
       if (this.isOnPrivateNote || this.isRichEditorEnabled) {
         return true;
       }
 
-      if (this.isAPIInbox) {
+      if (this.isAPIInbox(this.inbox)) {
```

Comment:
Let use directly from helper itself
```suggestion
      if (isAPIInbox(this.inbox)) {
```